storage-proofs/core/benches/pedersen.rs

```diff
@@ -157,60 +155,6 @@ fn pedersen_circuit_benchmark(c: &mut Criterion) {
     );
 }
 
-fn pedersen_md_circuit_benchmark(c: &mut Criterion) {
```

Comment:
Are you sure you want to remove this one? I'd rather add this to the `criterion_group!()` call below. You could check with the original author, if it was a mistake not adding it, or if it should be removed.

Reply:
Good point, @laser `git blame` highlights you as the developer who added the macro call `criterion_group`. Do you remember if there was a reason why `pedersen_md_circuit_benchmark()` was not added to it please?